client/packages/invoices/src/InboundShipment/DetailView/Footer/StatusChangeButton.tsx

```diff
@@ -20,121 +20,58 @@ import { InboundLineFragment, useInbound } from '../../api';
-      label: getButtonLabel(InvoiceNodeStatus.Delivered),
-      isDisabled: true,
-    },
-    {
-      value: InvoiceNodeStatus.Received,
-      label: getButtonLabel(InvoiceNodeStatus.Received),
-      isDisabled: true,
-    },
-    {
-      value: InvoiceNodeStatus.Verified,
-      label: getButtonLabel(InvoiceNodeStatus.Verified),
-      isDisabled: true,
-    },
-  ];
-
-  // User can only go forward through the statuses, so we enable the ones that are later in the order
-  // Should be refactored to make this easier to maintain, for some examples the next available status might not be as simple as this...
-
-  if (currentStatus === InvoiceNodeStatus.Shipped) {
-    // When the status is shipped the user can select from delivered, received and verified options
-    options[3].isDisabled = false;
-    options[4].isDisabled = false;
-    options[5].isDisabled = false;
-  }
-
-  if (currentStatus === InvoiceNodeStatus.Delivered) {
-    // When the status is Delivered, the user can select from received and verified options
-    options[4].isDisabled = false;
-    options[5].isDisabled = false;
-  }
-
-  if (currentStatus === InvoiceNodeStatus.Received) {
-    // When the status is Received, the user can only select verified option
-    options[5].isDisabled = false;
-  }
-
-  return options;
-};
-
-const getManualStatusOptions = (
-  currentStatus: InvoiceNodeStatus,
-  getButtonLabel: (status: InvoiceNodeStatus) => string
-): SplitButtonOption<InvoiceNodeStatus>[] => {
-  const options: [
-    SplitButtonOption<InvoiceNodeStatus>,
-    SplitButtonOption<InvoiceNodeStatus>,
-    SplitButtonOption<InvoiceNodeStatus>,
-    SplitButtonOption<InvoiceNodeStatus>,
-  ] = [
-    {
-      value: InvoiceNodeStatus.New,
-      label: getButtonLabel(InvoiceNodeStatus.New),
-      isDisabled: true,
-    },
-    {
-      value: InvoiceNodeStatus.Delivered,
-      label: getButtonLabel(InvoiceNodeStatus.Delivered),
-      isDisabled: true,
-    },
-    {
-      value: InvoiceNodeStatus.Received,
-      label: getButtonLabel(InvoiceNodeStatus.Received),
-      isDisabled: true,
-    },
-    {
-      value: InvoiceNodeStatus.Verified,
-      label: getButtonLabel(InvoiceNodeStatus.Verified),
-      isDisabled: true,
-    },
-  ];
-
-  if (currentStatus === InvoiceNodeStatus.New) {
-    // When the status is new, delivered and verified are available to
-    // select.
-    options[1].isDisabled = false;
-    options[2].isDisabled = false;
-    options[3].isDisabled = false;
-  }
-
-  // When the status is delivered, only verified is available to select.
-  if (currentStatus === InvoiceNodeStatus.Delivered) {
-    options[2].isDisabled = false;
-    options[3].isDisabled = false;
-  }
-
-  // When the status is delivered, only verified is available to select.
-  if (currentStatus === InvoiceNodeStatus.Received) {
-    options[3].isDisabled = false;
+  // Manual workflows skip Picked and Shipped statuses
+  const statuses = isManuallyCreated
```

Comment:
Did a bit of a refactor to consolidate the two manual/auto status options since they're essentially samesame

Reply:
yay